src/appComponents/sessions/SessionsController.js

```diff
@@ -19,26 +19,27 @@ sessionsModule.config(["$stateProvider", function($stateProvider) {
 sessionsModule.controller("SessionsController",
     ["$mdDialog", "$sessionStorage", "$rootScope", "ApiJsonFactory",
         function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-            var sc = this;
-            if ($sessionStorage.sessions === null ||
-                typeof ($sessionStorage.sessions) === "undefined")
-            {
-                $sessionStorage.sessions = [];
+
+          let sec = this;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected `this`.](https://www.codacy.com/app/triplez/open-event-webapp/file/2445549891/issues/source?bid=3233857&fileBranchId=3245777#l23)